app/v1/models/response/listPeriodSummaries/PeriodDetails.scala

```diff
@@ -19,16 +19,33 @@ package v1.models.response.listPeriodSummaries
 import play.api.libs.functional.syntax._
 import play.api.libs.json.{JsPath, Json, OWrites, Reads}
 
-case class PeriodDetails(periodId: String, periodStartDate: String, periodEndDate: String, periodCreationDate: Option[String])
+/*
+  Note that the field periodCreationDate is not returned by the downstream until Release 10 (R10). This code
+  should be reimplemented as part of the following ticket:
+
+  https://jira.tools.tax.service.gov.uk/browse/MTDSA-15576
+ */
+
+case class PeriodDetails(
+    periodId: String,
+    periodStartDate: String,
+    periodEndDate: String
+//    periodCreationDate: Option[String]
+)
 
 object PeriodDetails {
 
   implicit val reads: Reads[PeriodDetails] = (
     (JsPath \ "from").read[String] and
-      (JsPath \ "to").read[String] and
-      (JsPath \ "periodCreationDate").readNullable[String]
-  )((from, to, periodCreationDate) =>
-    PeriodDetails(periodId = s"${from}_$to", periodStartDate = from, periodEndDate = to, periodCreationDate = periodCreationDate))
+      (JsPath \ "to").read[String]
+//      (JsPath \ "periodCreationDate").readNullable[String]
```

Comment:
Could each of the comment-outs have a note next to it, ilke "// To be reinstated, see MTDSA-15595" ?

Reply:
(can also then search for the ticket number to find the bits to uncomment/feature-switch/whatevers)